catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/TestPreStoragePlugin.java

```diff
@@ -48,6 +49,8 @@
 
     private static final String GEO_NITF = "/i_3001a.ntf";
 
+    private static final String FILESYSTEM_INVALID_CHARS = "\\/:*?<>|";
```

Comment:
`&` and `;` are also bad characters. `.` as well. I'm not sure you care for a unit test, however.